apps/dashboard/components/logs/datetime/suggestions.tsx

```
@@ -0,0 +1,123 @@
+      aria-label="Time range options"
+    >
+      <ScrollArea className="w-full h-full rounded-md max-h-[390px]" ref={scrollAreaRef}>
+        <div className="flex flex-col gap-1.5 p-1">
+          {options.map(({ id, display, checked }, index) => (
+            <div
+              key={id}
+              className={cn("group relative w-full rounded-lg", "focus-within:outline-none")}
+            >
+              <button
+                type="button"
+                role="radio"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<input type="radio">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)